src/index.css

```diff
@@ -1,19 +1,11 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:wght@700&display=swap"');
+
 body {
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to reduce the space at the right of the book's card so the left and right white space are the same and the book cards are centered

![Screenshot_20230304_231417](https://user-images.githubusercontent.com/25227022/222930931-0ce7d138-f48e-4acc-9322-70bcd9fa9d01.png)